risc0/groth16/src/verifier.rs

```diff
@@ -22,8 +22,9 @@ use crate::{
     from_u256, g1_from_bytes, g2_from_bytes, ProofJson, PublicInputsJson, Seal, VerifyingKeyJson,
 };
 
-// Constants from: bonsai/ethereum/contracts/groth16/Groth16Verifier.sol
-// When running a new ceremony, update them by running cargo xtask bootstrap-groth16.
+// Constants from: risc0-ethereum/contracts/src/groth16/Groth16Verifier.sol
+// When running a new ceremony, update them by running cargo xtask bootstrap-groth16
+// after udpating the new Groth16Verifier.sol on the risc0-ethereum repo.
```

Comment:
Maybe a future improvement of this will be to save each ID to a file so we can update them more easily by running the bootstrap command rather than copy/paste. we could do `include_str!(file);` where file contains the id's themselves that get updated. I know a few places where we could do something like this like the reproducible build workflow so we could pile them up into a single gh issue.

Reply:
btw - no action needed on this, I'm just thinking out loud